Projects/templates/Projects/prs_vclosed.html

```diff
@@ -41,10 +41,15 @@
     </div>
 
     {% else %}
-    {% if page_user.profile.role == 'student' %}
-    <h3 style="color: white"> You have pull requests which are not yet verified :(. </h3>
+    {% if page_user.profile.role == 'student' %}<div class="bd">
```

Comment:
you have remove inline css, thats okay. But then if you are adding div and class bd, you need to define the properties of bd class in css file :smile: